content/getting-started.markdown

```diff
@@ -6,7 +6,7 @@ title: Getting started with the DNSimple API
 
 Welcome to the official [DNSimple](https://dnsimple.com/) Developer documentation.
 
-DNSimple provides DNS hosting and domain registration that is simple and friendly. We provide an extensive API and an easy-to-use web interface so you can get your domain registered and set up with a minimal amount of effort. The complete definition of our API is available in OpenAPIv3 [YAML](/v2/openapi.yml) and [JSON](/v2/openapi.json) formats.
+DNSimple provides DNS hosting and domain registration that is simple and friendly. We provide an extensive API and an easy-to-use web interface so you can get your domain registered and set up with a minimal amount of effort. The complete definition of our API is available in OpenAPIv3 [YAML](/v2/openapi.yml/) and [JSON](/v2/openapi.json/) formats.
```

Comment:
Are you sure about these changes? These are real URLs, with extensions. How can they be read as redirects? It's actually potentially an issue that there is a trailing slash as it tells the servers to look at folders.

```
curl -I https://developer.dnsimple.com/v2/openapi.yml
HTTP/2 200 
accept-ranges: bytes
age: 2
cache-control: public,max-age=0,must-revalidate
cache-status: "Netlify Edge"; fwd=miss
content-type: application/x-yaml
date: Wed, 21 May 2025 23:10:35 GMT
etag: "0476a1b6a07c54fe123ca7c7ac407d09-ssl"
server: Netlify
strict-transport-security: max-age=31536000
x-frame-options: DENY
x-nf-request-id: 01JVTHSTC5B70DVXSXJDBAZ9JJ
x-xss-protection: 1; mode=block
content-length: 203840
```

There's no redirect in here.

Reply:
you are correct, those are in fact not redirects, thank you for pointing this out.